index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <nav id="nav-bar"></nav>
+        <img id="header-img" src="Images/fruits..jpeg" alt=" This is a picture of fruits" />
+        <nav id="nav-bar" class="sticky">
+            <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how_it_works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>
+        </nav>
+        <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/S0S1M9bKlYk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    </header>
+
+    <div class="container"></div>
+    <section id="hero">
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input id="email" placeholder=" Please Write a Valid Email" type="email" name="email" required />
+
+            <input id="submit" type="submit" />
+        </form>
+    </section>
```

Comment:
The section tag should be in the div class container. You should write 👎 
Example: 
<div class="container">
<section id="hero">
        <form id="form" action="https://www.freecodecamp.com/email-submit">
            <input id="email" placeholder=" Please Write a Valid Email" type="email" name="email" required />

            <input id="submit" type="submit" />
        </form>
    </section>
</div>